group_vars/ezproxy/common.yml

```diff
@@ -0,0 +1,6 @@
+---
+shib_admin_user:
```

Comment:
where is this var being used/consumed?

Reply:
https://github.com/pulibrary/princeton_ansible/blob/444f7194df09861075e1c80376e79c53dabdac4f/roles/ezproxy/templates/shibuser.txt.j2#L16